### PR TITLE
docs: JR-574: Updates Service Registry CLI doc with PM+QE feedback

### PR DESCRIPTION
**Pull request by bredamc**

### CODE REVIEW

**Comment by jsenko** [comment]
Looks good! I have a few suggestions:
- In the Creating a Service Registry instance section, I would mention the `rhoas service-registry use` command, which is a nice way to select an existing instance into the current context. (Same in other Selecting a Service Registry instance to use sections)
- In the Creating a service account section, I'd mention the other two roles `admin` and `viewer` in addition to the `manager` and a short description about their capabilities, unless it's already mentioned elsewhere.
- Extend `or OpenAPI documents to define an API` to `or OpenAPI documents to define an REST API`?
- Optionally add a note that some of the steps in Uploading Service Registry artifacts can be shortened to a single command:
```
$ rhoas service-registry artifact create --type=AVRO --artifact-id=my-artifact https://raw.githubusercontent.com/redhat-developer/app-services-cli/main/docs/resources/avro-userInfo.json
Using default artifacts group.
Loading file from url: https://raw.githubusercontent.com/redhat-developer/app-services-cli/main/docs/resources/avro-userInfo.json
Artifact created
```
- Optionally add a note that many commands support switching to an alternative output format using `-o, --output string        Output format (json, yaml, yml)`.

**Comment by jsenko** [comment]
LGTM! One small note for the `rhoas service-registry use` command, you can run it bare without the `--name` argument, an it will give you a nice selection UI which is useful if you can't remember the instance name of are lazy to type:)

File: docs/registry/rhoas-cli-getting-started-registry/README.adoc [hunk 1]
```diff
@@ -145,20 +136,19 @@ $ rhoas service-registry create --name my-registry
...
+If you do not want to use the default values, enter the following command: `rhoas service-registry create`.
+You are prompted to enter the `Name` and optional `Description` for the {registry} instance.
 ====
 --
 
 . Verify that the {registry} instance is ready to use.
 +
 --
-.Reviewing details of a {registry} instance
+.Reviewing the details of the {registry} instance
```
Comment (smccarthy-ie) [inline docs/registry/rhoas-cli-getting-started-registry/README.adoc:151]:
```suggestion
.Reviewing the {registry} instance details
```

File: docs/registry/rhoas-cli-getting-started-registry/README.adoc [hunk 2]
```diff
@@ -169,8 +159,64 @@ because the `Status` field is `ready`.
...
+By default, many of the `rhoas` commands provide output in JSON format. You can use the `-o` option to specify a different output format. For more information, see the command help as described in {base-url}{getting-started-rhoas-cli-url-registry}#proc-commands-managing-registry_getting-started-rhoas-service-registry[Commands for managing {registry}].
+====
+--
+
+
+[id="proc-specifying-default-service-registry-instance-cli_{context}"]
+== Specifying the default {registry} instance
+
+[role="_abstract"]
+If you have multiple {registry} instances, you can configure a specific instance as the default {registry} instance.
```
Comment (smccarthy-ie) [inline docs/registry/rhoas-cli-getting-started-registry/README.adoc:171]:
```suggestion
If you have multiple {registry} instances, you can configure a default {registry} instance.
```

File: docs/registry/rhoas-cli-getting-started-registry/README.adoc [hunk 3]
```diff
@@ -414,23 +482,26 @@ $ rhoas service-registry artifact metadata-get --artifact-id=my-artifact
...
 
 [role="_abstract"]
-To prevent invalid content from being added to {registry}, you can configure optional rules to check the artifact content. Artifact rules apply to the specified artifact only. Global rules apply to all artifacts in a particular {registry} instance. Configured artifact rules override any configured global rules. Before a new artifact version can be uploaded to the registry, all configured global rules or artifact rules must pass.
+To prevent invalid content from being added to {registry}, you can configure optional rules to check the artifact content. Global rules apply to all artifacts in a particular {registry} instance. Artifact rules apply to the specified artifact only, and cannot be configured until the artifact is created. Configured artifact rules override any configured global rules.
+
+When you create an instance, the global rules are disabled by default. When you create an artifact, the artifact rules are disabled by default. To configure a global rule or artifact rule, you must first enable the rule. An artifact cannot be updated if the new content violates any of the enabled rules; in such cases, the registration fails.
```
Comment (smccarthy-ie) [inline docs/registry/rhoas-cli-getting-started-registry/README.adoc:495]:
```suggestion
When you create an instance, the global rules are disabled by default. When you create an artifact, the artifact rules are disabled by default. To configure a global rule or artifact rule, you must ensure that the rule is enabled. An artifact cannot be updated if the new content violates any of the enabled rules; in such cases, the registration fails.
```

**Review by smccarthy-ie — COMMENTED** [review]
Looks great, left a few small suggestions

File: docs/registry/rhoas-cli-getting-started-registry/README.adoc (hunk 1 above)
Comment (bredamc) [reply docs/registry/rhoas-cli-getting-started-registry/README.adoc]: Done, thanks!

File: docs/registry/rhoas-cli-getting-started-registry/README.adoc (hunk 3 above)
Comment (bredamc) [reply docs/registry/rhoas-cli-getting-started-registry/README.adoc]: Done, thanks!

**Review by smccarthy-ie — APPROVED** [review]
LGTM!

**Review by rkubis — APPROVED** [review]

**Pull request merged**